doc/concepts/concepts/events.rst

```diff
@@ -342,7 +342,7 @@ device for which at least one interrupt has been defined in the device tree.
 
 
 While the current SoC dtsi file, delivered in the
-`outpost-devicetree <https://github.com/outpost-os/outpost-devicetree>`_
```

Comment:
do we rename those repo too ?

Reply:
yes, and it has been already moved to camelot-os. I'm fixing that line